Application/src/main/AndroidManifest.xml

```diff
@@ -101,7 +101,8 @@
             </intent-filter>
         </activity>
 
-        <service android:name=".api.NotificationService">
+        <service android:name=".api.NotificationService"
+                 android:exported="false">
```

Comment:
:+1: 
Why do you need an IntentFilter here in the first place? You can just start the Service with its class right?